### DISCUSSION TITLE
v 0.42.3 notification

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#184] [General]
the uploading folders works ok, but the uploading folder containing illegal files does not throw a notification message that it has not been uploaded, this notification does not exist for illegal files either. a notification should be created that the deletion has not been performed.

**Comment by rejetto** [comment]
what you mean by illegal files?
please make an example of the notification you'd like to get.
if you use screenshots or video it will be easier to understand

**Comment by gogi100** [comment]
illegal file means, its extension is not found in 'accept in upload' on admin-panel.
if it is about files that is not found in 'accept in upload' on admin-panel, the message would be 'files were not uploaded because they are not allowed to upload'. if it is folder that contains illegal file or files 'the folder/the folders was not uploaded because it contains files that are not allowed to be uploaded'

**Comment by rejetto** [comment]
ok

**Comment by xrivo1** [comment]
Hello. Is it possible to insert an option that automatically bans for x time (for example 1 day) IP addresses that make fast calls to the site? maybe also setting the frequency (for example >30 calls per second) Thanks

**Comment by gogi100** [comment]
what does it mean when you say 'fast calls to the site?'

**Reply by xrivo1** [reply]
I refer to the fact that I found myself in the log file, 130 lines in 1 minute from the same IP address

**Comment by rejetto** [comment]
I guess you are talking about ANY kind of request, because 30 calls per second from the same IP sounds like a lot.
But anyway, let us know if I'm correct.

**Reply by xrivo1** [reply]
excuse my English but I'm Italian and I use google translate. I found some IP addresses that in 1 minute generated 120 lines in the log file. I don't know what kind of systems they use, maybe it's just a continuous PING, but it's a very disturbing thing. The 30 accesses per second was a simple example. I leave it to you experts to define the timing. In a nutshell it should be a system similar to the " block for X time after X failed login attempts "

**Comment by gogi100** [comment]
Yes, that's it.

**Comment by gogi100** [comment]
Maybe the solution without banning then limit the number of connections

**Reply by xrivo1** [reply]
Banning is another issue I've encountered. Being Italian, my IP is 52.*.*.*.  if in the ban I enter 2.*.*.* I can no longer access. In addition, with a simple VPN program you can get an Italian IP, so even bannabdo all the states of the world does not solve the problem :-)

**Comment by gogi100** [comment]
it's like some kind of ddos ​​attack. This option in the hfs is ok. Also
connrctions number can load the link, like the torrent

**Comment by rejetto** [comment]
i'm italian too.
I think I will make a plugin for this autoban.
Anyway, we are writing in this page that's about a different topic.
This deserved a new discussion page.

**Reply by xrivo1** [reply]
Ok. I apologize for the offtopic.  I am happy that such a beautiful and functional program was written by a compatriot. :-)

**Comment by gogi100** [comment]
Ok
